app/assets/javascripts/swagger-ui/threescale.coffee.erb

```diff
@@ -40,10 +40,13 @@ class ThreeScaleHttp
   constructor: (swagger_http) ->
     @swagger_http = swagger_http
 
+  enableApiDocsProxy: ->
```

Comment:
Not major, and less even since this is legacy and hopefully soon to be removed... but, since this is set by config and not a method that enables/disables something, I'd name it `isApiDocsProxyEnabled`